EnemizerCLI.Core/Program.cs

```diff
@@ -13,27 +13,27 @@ class Program
     {
         static void Main(string[] args)
         {
-            var options = new CommandLineOptions();
-            if (CommandLine.Parser.Default.ParseArgumentsStrict(args, options))
+	        var writer = new StringWriter();
+            var parser = new Parser(conf => conf.HelpWriter = writer);
+            var result = parser.ParseArguments<CommandLineOptions>(args);
+
+            result.WithNotParsed(error =>
             {
-                Stopwatch stopwatch = new Stopwatch();
-                if (options.BinaryMode)
-				{
-                    stopwatch.Start();
-                    MakeEnemizerRom(options);
-				}
-                else
-				{
-                    stopwatch.Start();
-                    MakeEnemizerJsonPatch(options);
-                }
-                stopwatch.Stop();
-                Console.WriteLine($"Seed generated in: {stopwatch.Elapsed}");
-            }
-            else
+				Console.WriteLine(writer.ToString());
+				Environment.Exit(1);
```

Comment:
Tabs here should probably be spaces.

This is the last message I'll put up about tabs, since that'll get tiring quick. A quick conversion over all the affected files should be quick and easy in VS. Let me know if it poses any problem.